package.json

```diff
@@ -39,14 +41,14 @@
 		"svelte-preprocess": "^5.0.4",
 		"ts-jest": "^29.1.1",
 		"tslib": "^2.6.2",
-		"eslint": "^8.49.0",
-		"typescript": "^5.2.2"
+		"typescript": "^5.2.2",
+		"copyfiles": "^2.4.1",
+		"@types/lz-string": "^1.5.0"
```

Comment:
this probably didn't mean to be in build?

Reply:
npm WARN deprecated @types/lz-string@1.5.0: This is a stub types definition. lz-string provides its own type definitions, so you do not need this installed.

lol i guess it shouldn't be there anyway

Reply:
removed it